assembl/static2/css/components/fictionSideComment.scss

```diff
@@ -0,0 +1,82 @@
+.side-comment-badge {
```

Comment:
I thought we decided to put the css file in the same folder than the component?

Reply:
So have we started to do this yet ?

Reply:
Not yet. Closing